content/2_3_riak.tex

```diff
@@ -0,0 +1,58 @@
+\chapter{The Riak Key-Value Store}
+\chapterauthor{Daniel Rutz and Paul Thore Flachbart}
```

Comment:
The Neo4J chapter used commas here, so I guess everyone else should too?  Not quite sure.

Reply:
Agree.